README.md

```diff
@@ -82,6 +82,20 @@ export MM_ADMIN_USERNAME=admin
 export MM_ADMIN_PASSWORD=password
 ```
 
+## Troubleshooting
+
+### Error message: `x509: certificate signed by unknown authority`
```

Comment:
This issue is not specific to this repo. It has come up e.g. for Matterpoll. Maybe it's worth to add a "Plugin Troubleshooting" section to the MM docs?